platform/ios/ios.xcodeproj/project.pbxproj

```diff
@@ -1335,6 +1333,15 @@
 			path = "Integration Test Harness";
 			sourceTree = "<group>";
 		};
+		1F432781203B74DF00A21E6D /* Recovered References */ = {
```

Comment:
The Recovered References group indicates that there’s a problem with the membership of the files it contains.

> When a project is opened, Xcode may create a Recovered References group in the Project navigator. This group contains files that are referenced in a target but aren't referenced in the group tree of the project. If the group appears, move each file to the proper location in the project as appropriate.